### DISCUSSION TITLE
Roadmap for Catalog in US Gov Clouds?

### DISCUSSION THREAD

**Discussion by jwillisrose** [microsoft/PowerPlatform-Catalog#23] [Q&A]
Is there a roadmap for Power Platform Catalog to be made available in the US Government clouds?  (i.e. GCC, GCC-High, DoD).

Related q: will the solution file be available for customers who can't use App Source (e.g. US government customers)?

**Comment by MattB-msft** [comment]
@derekkwanpm for the answer here.

**Comment by derekkwanpm** [comment]
You will be able to install Catalog from PPAC, but not until sometime in June. Will reply back to this when available.

-Derek
________________________________

**Reply by dpardpar** [reply]
Do you happen to know what this account is used for that has administrative privileges? CatalogServiceDod; CatalogServiceDod@onmicrosoft.com. I have a customer going through some DoD ATO paperwork and inquired with me, along with the ones below:
# CatalogServiceDod; [CatalogServiceDod@onmicrosoft.com](mailto:CatalogServiceDod@onmicrosoft.com)
# CDSUserManagementApi; [CDSUserManagementApi@onmicorsoft.com](mailto:CDSUserManagementApi@onmicorsoft.com)
# DynamicsInstaller; [DynamicsInstaller@onmicrosoft.com](mailto:DynamicsInstaller@onmicrosoft.com)
# JobsServiceDOD; [JobsServiceDOD@onmicrosoft.com](mailto:JobsServiceDOD@onmicrosoft.com)
# PowerPlatformEnvironmentManagementDod; [PowerPlatformEnvironmentManagementDod@onmicrosoft.com](mailto:PowerPlatformEnvironmentManagementDod@onmicrosoft.com)

**Reply by jwillisrose** [reply]
> You will be able to install Catalog from PPAC, but not until sometime in June. Will reply back to this when available.
> […](#)
> -Derek
> ________________________________ From: MattB ***@***.***> Sent: Tuesday, May 7, 2024 9:00 AM To: microsoft/PowerPlatform-Catalog ***@***.***> Cc: Derek Kwan ***@***.***>; Mention ***@***.***> Subject: Re: [microsoft/PowerPlatform-Catalog] Roadmap for Catalog in US Gov Clouds? (Discussion #23) @derekkwanpm<https://github.com/derekkwanpm> for the answer here. — Reply to this email directly, view it on GitHub<[#23 (comment)](https://github.com/microsoft/PowerPlatform-Catalog/discussions/23#discussioncomment-9343818)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2PB5B3PNXFUN6KG5HF5GWTZBD3BLAVCNFSM6AAAAABHJMNERWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGNBTHAYTQ>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Any update on the Catalog in various US sovereign clouds?   Have DoD customer interested in leveraging it.

**Comment by derekkwanpm** [comment]
Hello, sorry for the delay. These are service identities used by Microsoft  Platform Services to access environments for various purposes, such as application deployment. Apologies, they should be documented on this page: Special system users and application users - Power Platform | Microsoft Learn<https://learn.microsoft.com/en-us/power-platform/admin/system-application-users>
[https://learn.microsoft.com/en-us/media/open-graph-image.png]<https://learn.microsoft.com/en-us/power-platform/admin/system-application-users>
Special system users and application users - Power Platform<https://learn.microsoft.com/en-us/power-platform/admin/system-application-users>
Learn about the special system and application users created when the system is provisioned, including assigned security role, user name, and purpose.
learn.microsoft.com


-Derek
________________________________

**Comment by derekkwanpm** [comment]
Hi Jamie, we're targeting mid-Oct for support for GCC, will keep you posted as we get a more precise timeframe.

-Derek
________________________________

**Reply by matthew-winter449** [reply]
Hi Derek,

Is there any update on the timing for GCC support. I have a few very interested customers.

Thanks!

**Comment by ghost** [comment]
@derekkwanpm or @MattB-msft , do either of you have an update on this? The last message said OCT2024, but I still don't see the Catalog in GCC

**Reply by Grayson-Bishop** [reply]
I had to move my account, but this "ghost" is me! Still would love an update on this
